styles/main.less

```diff
@@ -1,41 +1,28 @@
 @import "ui-variables";
 @import "syntax-variables";
 
-.smalls.round-box {
-  box-sizing: border-box;
-  border-radius: @component-border-radius;
-}
 
 @color-error-lighten: lighten(@background-color-error, 30%);
 
+@smalls-label-color: hsl( hue( @text-color-warning ), 100% - saturation( @base-background-color ), 60% - lightness( @app-background-color ) * 0.2 );
 atom-text-editor, atom-text-editor::shadow {
   .smalls-label {
-    .smalls.round-box;
-    // border: 1px solid contrast(@background-color-info);
-    // color: contrast(@background-color-info);
-    // background-color: @background-color-info;
-    // box-shadow: 0 0 2px contrast(@background-color-info);
-
+    z-index   : 1;
+    color     : @smalls-label-color;
+    margin-top: -1.5rem;
```

Comment:
To perfectly overlay labels on text, the value of `margin-top: -1.5rem;` needs to be slightly tuned for different font and font-size. Do you have any better idea to do this?